packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2720,9 +2712,9 @@ class _StandardBottomSheetState extends State<_StandardBottomSheet> {
     assert(widget.animationController == oldWidget.animationController);
   }
 
-  Future<void> close() async {
+  void close() {
```

Comment:
This was the reason for the google3 failure: In #67078 this method was marked `async`, which changes the timing slightly for tests. There's no need for it to be async, though. The main goal of this commit is to revert that.